test/production/eslint-plugin-deps/index.test.ts

```diff
@@ -82,6 +82,44 @@ describe('eslint plugin deps', () => {
       { "allowComparingNullableBooleansToTrue": false }
     ]
   }
+}
+        `,
+        'tsconfig.json': `{
+  "compilerOptions": {
+    "target": "ES2017",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
+    "strictNullChecks": true,
+    "noEmit": true,
+    ${/* The rule @typescript-eslint/no-unnecessary-boolean-literal-compare requires the `strictNullChecks` compiler option to be turned on to function correctly. */ ''}
```

Comment:
tsconfigs allow comments so no need to use `${''}`. The comment is also misplaced.